src/Sylius/Bundle/FlowBundle/Process/Builder/ProcessBuilder.php

```diff
@@ -79,6 +79,10 @@ public function add($name, $step)
             throw new \InvalidArgumentException('Step added via builder must implement "Sylius\Bundle\FlowBundle\Process\Step\StepInterface"');
         }
 
+        if (false === $step->isActive()) {
```

Comment:
If think, that now both ``isActive()`` implementations ([this](https://github.com/Sylius/Sylius/blob/f5d1b5f7fe845adfcc2fac485d6e19cd1708e03b/src/Sylius/Bundle/FlowBundle/Process/Step/AbstractStep.php#L59) and [this](https://github.com/Sylius/Sylius/blob/f5d1b5f7fe845adfcc2fac485d6e19cd1708e03b/src/Sylius/Bundle/FlowBundle/Process/Step/AbstractControllerStep.php#L60)) returns hardcoded ``true`` 😄